angular.json

```diff
@@ -170,12 +170,32 @@
               "localize": ["std"]
             },
             "development": {
+              "fileReplacements": [
+                {
+                  "replace": "src/environments/environment.ts",
+                  "with": "src/environments/environment.dev.ts"
+                }
+              ],
+              "optimization": true,
+              "outputHashing": "all",
+              "sourceMap": false,
+              "namedChunks": false,
+              "extractLicenses": true,
+              "vendorChunk": false,
+              "buildOptimizer": true,
```

Comment:
Are these changes necessary because of the updates to the dependency versions?

Reply:
The issue was that not always user idle was showing up. I created a "prod build" for develop to test it. I can revert it, to still have easier to debug, but not optimised version on dev env.